skeleton/cli-bundler/aurelia_project/tasks/watch.ext

```diff
@@ -12,17 +12,62 @@ import * as gulpWatch from 'gulp-watch';
 import * as debounce from 'debounce';
 import * as project from '../aurelia.json';
 // @endif
+// @if feat.plugin
```

Comment:
Please don't reuse this watch task.
This watch task is to watch both dev-app/ and src/ folders, and reload browser window for `au run`.

It's better to implement build-plugin using standard gulp.watch, it's quite simple.

In your build-plugin task
```js
if (CLIOptions.hasFlag('watch')) {
  main = function() {
    console.log('Watching and building plugin sources ...');
// need to adjust extnames based on app feature selection.
    return gulp.watch('src/**/*.{js,html,scss}', {ignoreInitial: false}, build);
  };
} else {
  main = gulp.series(
    build,
    () => console.log(doneMessage)
  );
}
```